ext/semian/resource.h

```diff
@@ -21,7 +21,7 @@ int system_max_semaphore_count;
  * Creates a new Resource. Do not create resources directly. Use Semian.register.
  */
 VALUE
-semian_resource_initialize(VALUE self, VALUE id, VALUE tickets, VALUE permissions, VALUE default_timeout);
+semian_resource_initialize(VALUE self, VALUE id, VALUE tickets, VALUE quota, VALUE permissions, VALUE default_timeout);
```

Comment:
Yeah, I don't think the name matters _that_ much since it's really internal to the Semian resource. In fact, this should probably be defined with `rb_define_protected_method`.